### PR TITLE
Change from TravisCI to GitHub Actions

### PR DESCRIPTION
**Pull request by benformosa**
GitHub actions seem to run faster than Travis, plus Travis has implemented limitations for OSS users: https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

I notice that your molecule configuration seems to be adapted from Jeff Geerling's, I've adapted his GitHub Actions workflows for this role.

This PR includes changing to test on the latest versions of most of the underlying tools. This includes python 3, molecule 3, ansible 2.10 and testinfra 6.
Molecule is run against both CentOS 7 and 8.

This PR also includes a commit which changes one of the tests. This was required to get the tests to run successfully.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        with:
+          path: 'theforeman.foreman_scap_client'
+
+      - name: Set up Python 3.
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install test dependencies.
+        run: pip3 install ansible molecule[docker] docker testinfra
```
Comment (xprazak2) [inline .github/workflows/ci.yml:60]: The reason for not using requirements.txt is that we only need specific dependencies and not all of them for given step, correct?

**Comment by xprazak2** [comment]
Nice, I was thinking we would need to move away from Travis at some point. I found one issue - the Lint is green even though there are fatal errors: https://github.com/xprazak2/ansible-foreman_scap_client/runs/1486042480?check_suite_focus=true

File: .github/workflows/ci.yml (hunk 1 above)
Comment (benformosa) [reply .github/workflows/ci.yml:60]: I've copied this pretty much directly from [geerlingguy/ansible-role-drupal](https://github.com/geerlingguy/ansible-role-drupal/blob/master/.github/workflows/ci.yml).
Looks like I didn't end up committing a changed `requirements.txt`.

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**